node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/PeriodicApplicationMaintainer.java

```diff
@@ -60,6 +64,11 @@ protected boolean shouldBeDeployedOnThisServer(ApplicationId application) {
         return deployer().lastDeployTime(application).isPresent();
     }
 
+    // TODO: Do not start deploying until some time has gone (ideally only until bootstrap of config server is finished)
+    protected boolean waitInitially() {
+        return Instant.now().isBefore(start.plus(minTimeBetweenRedeployments));
```

Comment:
What is the default value of minTimeBetweenRedeployments ?

Reply:
30 minutes

Reply:
It can't figure when it is safe ?
Is it a bit long ? Perhaps 2 minutes  is sufficient ?
Will it cause issues for tests ?

Reply:
Discussed offline. I'll look into if the maintainer can ask if bootstrap is finished instead.